suite-native/navigation/src/components/TabBar.tsx

```diff
@@ -50,7 +51,7 @@ export const TabBar = ({ state, navigation, tabItemOptions }: TabBarProps) => {
 
                     if (!isFocused && !event.defaultPrevented) {
                         // The `merge: true` option makes sure that the params inside the tab screen are preserved
-                        navigation.navigate(route.name, { merge: true });
+                        navigation.navigate(routeName, { ...params, merge: true });
```

Comment:
Do we want to merge? I think for our usage will be better not to merge params, but not sure how this works exactly.

Reply:
I can't remember exactly why this is here. But as I understand this behaviour (this is a [good explanation](https://stackoverflow.com/a/67327737), this is not wanted => removed 3b74ca6b24ac779493213183de00588f27fbf0b6.